### ISSUE TITLE
Deprecated in WooCommerce 3.0: WC_Shipping_Zone::get_zone_id, use WC_Shipping_Zone::get_id instead.

### ISSUE THREAD

**Issue by ianfoxofficial** [hyyan/woo-poly-integration#130]
Got this error message since updating to WooCommerce 3.0:

Notice: WC_Shipping_Zone::get_zone_id is verouderd sinds versie 3.0. Gebruik in plaats daarvan WC_Shipping_Zone::get_id. in C:\HostingSpaces\BandJ\staging.mycosycompany.eu\wwwroot\wp-includes\functions.php on line 3829

I have 3 Shipping zones so this line is repeated 3 times.

I'm lucky this is on dev. :) 

Thanks for your plugin! 👍

**Comment by theleemon** [comment]
Try this to avoid some notices with WC 3.0.:

In woo-poly-integration/src/Hyyan/WPI/Shipping.php file, line 138 to 140 must be :

```
        $zones[ $zone->get_id() ] = $zone->get_data();
        $zones[ $zone->get_id() ]['formatted_zone_location'] = $zone->get_formatted_location();
        $zones[ $zone->get_id() ]['shipping_methods'] = $zone->get_shipping_methods();

```

**Comment by ianfoxofficial** [comment]
Thank you! :)

**Comment by ianfoxofficial** [comment]
I've tested it some more, there are a lot of extra problems when having the plugin active.

**Comment by theleemon** [comment]
I suppose every reference to old properties must be changed to use the new CRUD methods in WooCommerce 3.0:

https://github.com/woocommerce/woocommerce/wiki/Product-Data-Schema

Hope the plugin dev updates the code.

**Comment by SillyMe727** [comment]
I am receiving the same errors. I am hoping for some support/ update of the plugin soon.

**Comment by Jon007** [comment]
Fixed in pull request 0.30RC1, the code is available at:
https://github.com/Jon007/woo-poly-integration

This should be treated as beta software until we have completed the testing, merge and release.

It would be great if you can test it!!!
If you find any issues with the RC, please report as separate issues, I will fix asap.
